pycbc/inference/models/marginalized_gaussian_noise.py

```diff
@@ -360,6 +360,11 @@ def _loglr(self):
                                                                  params['tc'])
             dtc = params['tc'] + dt
 
+            pol_phase = numpy.exp(-2.0j * params['polarization'])
```

Comment:
This should only be in the case of precalculated values. Otherwise, it will actuall mess up the polarization values.